website/src/developing-extensions/class-reflection-extensions.md

```diff
@@ -24,7 +24,7 @@ interface PropertiesClassReflectionExtension
 }
 ```
 
-Most likely you will also have to implement a new class implementing the `PropertyReflection` interface:
+Most likely you will also have to implement a new class implementing the [`PropertyReflection`](https://github.com/phpstan/phpstan-src/blob/master/src/Reflection/PropertyReflection.php) interface:
 
 ```php
```

Comment:
I am wondering that the list of methods here does not reflect the interface in the source repo

Reply:
`interface PropertyReflection extends ClassMemberReflection` so I added methods from the parent too.